src/pymodaq_gui/managers/action_manager.py

```diff
@@ -13,12 +13,17 @@
 
 def create_icon(icon_name: str):
     icon = QtGui.QIcon()
-    if Path(icon_name).is_file():
+    if Path(icon_name).is_file(): # Test if icon is in path
         icon.addPixmap(QtGui.QPixmap(icon_name), QtGui.QIcon.Normal,
                        QtGui.QIcon.Off)
     else:
-        icon.addPixmap(QtGui.QPixmap(f":/icons/Icon_Library/{icon_name}.png"), QtGui.QIcon.Normal,
-                       QtGui.QIcon.Off)
+        pixmap = QtGui.QPixmap(f":/icons/Icon_Library/{icon_name}.png") # Test if icon is in pymodaq's library
+        if pixmap.isNull(): 
+            if hasattr(QtGui.QIcon.ThemeIcon,icon_name): # Test if icon is in Qt's library
```

Comment:
this would cause an error with Qt<6.7 because the ThemeIcon module didn't exist! So you should also test if ThemeIcon is present, or just use a try/except statement both for ThemeIcon presence and the icon_name and catch the AttributeError error